Watchman.DomainModel/Settings/Configuration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings
+{
+    public class Configuration : Entity, IAggregateRoot
+    {
+        public int UserMessagesCountToBeSafe { get; private set; }
+
+        internal static Configuration GetDefaultConfiguration()
+        {
+            return new Configuration
+            {
+                UserMessagesCountToBeSafe = 500
```

Comment:
tutaj zamiast sztywnej liczby chyba lepiej zastosować coś innego
coś co byłoby oparte o to jak długo ktoś jest na serwerze
i jak często sie wypowiada

bezpieczna osoba to powinien być regularny użytkownik
a nie ktoś kto kiedyś wszedł i naklepał X wiadomości na raz